fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/utils/impl/DistributedFixityCheck.java

```diff
@@ -0,0 +1,117 @@
+/**
+ * Copyright 2015 DuraSpace, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.fcrepo.kernel.impl.utils.impl;
```

Comment:
Wasn't this file removed a while ago? Certainly the package name `org.fcrepo.kernel.impl` shouldn't be there.